<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ameet" failures="18" tests="18" name="Test" time="63.878" errors="0" timestamp="2024-12-17T19:45:09 IST">
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getAllStocksTest" time="0.125">
    <failure type="java.lang.AssertionError" message="getAllStocks must be implemented using Rest Assured methods only. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getAllStocks must be implemented using Rest Assured methods only. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getAllStocksTest(RestAssured_TestCases_PL1.java:36)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getAllStocksTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getMainStoreTest" time="0.051">
    <failure type="java.lang.AssertionError" message="getMainStore must be implemented using Rest Assured methods only. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getMainStore must be implemented using Rest Assured methods only. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getMainStoreTest(RestAssured_TestCases_PL1.java:72)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getMainStoreTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getRequisitionByDateRangeTest" time="0.011">
    <failure type="java.lang.AssertionError" message="getRequisitionByDateRange must be implemented using Rest Assured methods only. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getRequisitionByDateRange must be implemented using Rest Assured methods only. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getRequisitionByDateRangeTest(RestAssured_TestCases_PL1.java:110)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getRequisitionByDateRangeTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getPatientConsumptionsTest" time="0.008">
    <failure type="java.lang.AssertionError" message="getPatientConsumptions must be implemented using Rest Assured methods only. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getPatientConsumptions must be implemented using Rest Assured methods only. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getPatientConsumptionsTest(RestAssured_TestCases_PL1.java:153)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getPatientConsumptionsTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getPatientConsumptionInfoTest" time="0.012">
    <failure type="java.lang.AssertionError" message="getPatientConsumptionInfoByPatientIdAndVisitId must be implemented using Rest Assured methods only. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getPatientConsumptionInfoByPatientIdAndVisitId must be implemented using Rest Assured methods only. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getPatientConsumptionInfoTest(RestAssured_TestCases_PL1.java:194)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getPatientConsumptionInfoTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getBillingSchemeBySchemeIdTest" time="0.016">
    <failure type="java.lang.AssertionError" message="getBillingSchemeBySchemeId method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getBillingSchemeBySchemeId method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getBillingSchemeBySchemeIdTest(RestAssured_TestCases_PL1.java:236)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getBillingSchemeBySchemeIdTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getBillingSummaryByPatientIdTest" time="0.03">
    <failure type="java.lang.AssertionError" message="getBillingSummaryByPatientId method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getBillingSummaryByPatientId method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getBillingSummaryByPatientIdTest(RestAssured_TestCases_PL1.java:262)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getBillingSummaryByPatientIdTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getConsumptionsListOfAPatientByIdTest" time="0.013">
    <failure type="java.lang.AssertionError" message="getConsumptionsListOfAPatientById method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getConsumptionsListOfAPatientById method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getConsumptionsListOfAPatientByIdTest(RestAssured_TestCases_PL1.java:291)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getConsumptionsListOfAPatientByIdTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getReturnConsumptionsListTest" time="0.009">
    <failure type="java.lang.AssertionError" message="getReturnConsumptionsList method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getReturnConsumptionsList method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getReturnConsumptionsListTest(RestAssured_TestCases_PL1.java:318)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getReturnConsumptionsListTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getDischargedPatientsTest" time="0.007">
    <failure type="java.lang.AssertionError" message="getDischargedPatients method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getDischargedPatients method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getDischargedPatientsTest(RestAssured_TestCases_PL1.java:355)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getDischargedPatientsTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getAdmittedPatientsTest" time="0.015">
    <failure type="java.lang.AssertionError" message="getAdmittedPatients method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getAdmittedPatients method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getAdmittedPatientsTest(RestAssured_TestCases_PL1.java:394)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getAdmittedPatientsTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="searchIpdPatientByPatientIdTest" time="0.005">
    <failure type="java.lang.AssertionError" message="searchIpdPatientByPatientId method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: searchIpdPatientByPatientId method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.searchIpdPatientByPatientIdTest(RestAssured_TestCases_PL1.java:437)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- searchIpdPatientByPatientIdTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getPatientProvisionalInfoTest" time="0.011">
    <failure type="java.lang.AssertionError" message="getPatientProvisionalInfo method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getPatientProvisionalInfo method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getPatientProvisionalInfoTest(RestAssured_TestCases_PL1.java:464)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getPatientProvisionalInfoTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getProvisionalItemsListByPatientIdAndSchemeIdTest" time="0.007">
    <failure type="java.lang.AssertionError" message="getProvisionalItemsListByPatientIdAndSchemeId method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getProvisionalItemsListByPatientIdAndSchemeId method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getProvisionalItemsListByPatientIdAndSchemeIdTest(RestAssured_TestCases_PL1.java:496)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getProvisionalItemsListByPatientIdAndSchemeIdTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getInvoicesByDateRangeTest" time="0.005">
    <failure type="java.lang.AssertionError" message="getInvoicesByDateRange method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getInvoicesByDateRange method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getInvoicesByDateRangeTest(RestAssured_TestCases_PL1.java:533)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getInvoicesByDateRangeTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getProviderListTest" time="0.005">
    <failure type="java.lang.AssertionError" message="getProviderList method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getProviderList method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getProviderListTest(RestAssured_TestCases_PL1.java:565)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getProviderListTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getUsersListTest" time="0.009">
    <failure type="java.lang.AssertionError" message="Status code should be 200. expected [200] but found [0]">
      <![CDATA[java.lang.AssertionError: Status code should be 200. expected [200] but found [0]
at testcases.RestAssured_TestCases_PL1.getUsersListTest(RestAssured_TestCases_PL1.java:600)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- getUsersListTest -->
  <testcase classname="testcases.RestAssured_TestCases_PL1" name="getCurrentFiscalYearDetailsTest" time="0.006">
    <failure type="java.lang.AssertionError" message="getCurrentFiscalYearDetails method validation failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: getCurrentFiscalYearDetails method validation failed. expected [true] but found [false]
at testcases.RestAssured_TestCases_PL1.getCurrentFiscalYearDetailsTest(RestAssured_TestCases_PL1.java:634)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getCurrentFiscalYearDetailsTest -->
</testsuite> <!-- Test -->
